README.md

```diff
@@ -1,2 +1,40 @@
-# web-testeexec-navigator
+# web-testexec-navigator
+
 Test-Editor test execution navigator gui angular component
+
+# TestexecNavigator
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.7.4.
+
```

Comment:
Prerequisites for the following steps are 
* an initial run of `npm install` / `yarn install` (right?)
* either a global installation of Angular CLI (**ng**), or using the local one downloaded by the previous step
    * how about a symlink to `node_modules/@angular/cli/bin/ng` in the base directory?